Trackers/iou_tracker/iou.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+from ..track import Track
+from scipy.optimize import linear_sum_assignment as linear_assignment
+
+class iou_track(Track):
+    def __init__(self, id, bbox, hits, miss):
+        Track.__init__(self, id, bbox, hits, miss)
+
+class iou_tracker():
+    def __init__(self):
+        self.tracks = []
+        self.nextId = 0
+        self.max_lost=10
```

Comment:
Do not unnecessarily change the variable name, we will maintain the uniform naming throughout the code base.